src/Settings/LocalFactory.php

```diff
@@ -70,17 +70,17 @@ public function __construct(RemoteSettingsRepository $remoteRepository)
     public function create(SearchIndex $index, string $model): Settings
     {
         $attributes = $this->getAttributes($model);
-        $attributesArray = [];
+        $arrayFillKeys = [];
```

Comment:
The comment was for *using* the array_fill_keys function. Not rename the variable.

Reply:
sorry, my bad. it's done.